nautobot/extras/migrations/0087_joblogentry__log_level_data_migration.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations
+
+
+LOG_LEVEL_MAPPING = {
+    "default": "info",
+    "success": "info",
+    "failure": "error",
+}
+
+
+def migrate_job_log_entry_log_levels(apps, schema_editor):
+    """
+    Migrate job log levels from 1.0 values to 2.0 values.
+    default => info
```

Comment:
I think default was rendered as a debug message on the job result page. The new default is `info` but should it be `debug` instead? If we migrate `default` to `debug` it would make it possible to reverse the migration.

Reply:
I think it's fine for the `log()` API to now default to `info`, but I do think it would be more correct to past usage to map `default` entries to `debug`.